src/brevitas/graph/gpxq.py

```diff
@@ -232,7 +238,8 @@ def process_input(self, inp):
         if isinstance(inp, IntQuantTensor):
             if is_quant_enabled and self.quant_metadata is None:
                 self.quant_metadata = _CachedIO(inp, metadata_only=True)
-            inp = inp.value
+            if isinstance(inp, QuantTensor):
```

Comment:
When could this not be `True`? I see you've added this check, but it's not obvious to me. Can `inp` get modified in-place by `_CachedIO`?

Reply:
I think the answer to my question is "no", so I've removed that check in [`1f0933f`](https://github.com/Xilinx/brevitas/pull/1013/commits/1f0933f472480659e2b1ece2908728da969a998d).